build/build.py

```diff
@@ -485,8 +485,8 @@ def generator(self):
     @property
     def generate_options(self):
         return super().generate_options + [
-                f'-DCMAKE_TOOLCHAIN_FILE={get_vcpkg_toolchain_file_path(self.vcpkg_root_path)}',
-                f'-DVCPKG_TARGET_TRIPLET={self._vcpkg_triplet}'
+                '-DCMAKE_TOOLCHAIN_FILE={}'.format(get_vcpkg_toolchain_file_path(self.vcpkg_root_path)),
+                '-DVCPKG_TARGET_TRIPLET={}'.format(self._vcpkg_triplet)
```

Comment:
Was there an issue with the f string?

Reply:
Can't remember but is it because a version of python within arm builds is too old to support fstrings?

Reply:
nit: fix commented lines on 498-502, or consider removing them

Reply:
Yeah deb9 python is too old to support it.